### PR TITLE
chore: add more aggregate functions to benchmark test

### PR DESCRIPTION
**Pull request by huaxingao**
## Which issue does this PR close?

Add `

Closes #.

## What changes are included in this PR?

Add `MIN`, `MAX`, `COUNT` to AggregateBenchmark. I currently have an issue with `AVG` for the `Grouped Aggregate (multiple group keys + single aggregate)`, so I haven't included `AVG` yet.

### CODE REVIEW

File: spark/benchmarks/CometAggregateBenchmark-jdk11-results.txt [hunk 1]
```diff
@@ -0,0 +1,464 @@
+================================================================================================
```
Comment (kazuyukitanimura) [inline spark/benchmarks/CometAggregateBenchmark-jdk11-results.txt:1]: We don't have other bench results under `spark/benchmarks`, so we may not need to commit this file...

**Review by kazuyukitanimura — APPROVED** [review]

File: spark/benchmarks/CometAggregateBenchmark-jdk11-results.txt (hunk 1 above)
Comment (andygrove) [reply spark/benchmarks/CometAggregateBenchmark-jdk11-results.txt:1]: These result files are very specific to the hardware they are run on, so I'm not sure it is worth checking these in.

Comment (huaxingao) [reply spark/benchmarks/CometAggregateBenchmark-jdk11-results.txt]: deleted

**Review by andygrove — APPROVED** [review]
LGTM. Thanks @huaxingao

**Comment by huaxingao** [comment]
Merged. Thanks @andygrove and @kazuyukitanimura for reviewing!

**Pull request merged**